cime_config/testmods_dirs/allactive/wcprodssp/user_nl_eam

```diff
@@ -1,7 +1,9 @@
  nhtfrq =   -24,-24,-6,-6,-3,-24,-24
  mfilt  = 1,30,120,120,240,30,1
  avgflag_pertape = 'A','A','I','A','A','A','I'
- fexcl1 = 'CFAD_SR532_CAL', 'LINOZ_DO3', 'LINOZ_DO3_PSC', 'LINOZ_O3CLIM', 'LINOZ_O3COL', 'LINOZ_SSO3', 'hstobie_linoz'
+!temporarily remove LINOZ O3 related fields from the list until updated linoz v3 style inputdata is ready
+!fexcl1 = 'CFAD_SR532_CAL', 'LINOZ_DO3', 'LINOZ_DO3_PSC', 'LINOZ_O3CLIM', 'LINOZ_O3COL', 'LINOZ_SSO3', 'hstobie_linoz'
```

Comment:
Note that this is "fexcl" which is itself trying to remove these variables from output.  The error was because the variables were never there to begin with.  So commenting on the command to exclude them is correct.